e2e/next-sandbox/pages/offline.tsx

```diff
@@ -1,5 +1,5 @@
 import { LiveList } from "@liveblocks/client";
-import type { IWebSocketCloseEvent } from "@liveblocks/core";
+import { INTERNAL } from "@liveblocks/core";
```

Comment:
Love the change to symbols for this! I just checked out the `.d.ts` file and I think it's a great solution — glad you found this way 👍 

One small suggestion: when I suggested the `kInternal` naming convention, it was the export/variable that I meant, not the string assigned to the Symbol() function (that's only used for debug logging).

We could define it like so:

```ts
export const kInternal = Symbol('liveblocks internals');
```

or even just

```ts
export const kInternal = Symbol();
```

Reply:
Updated!